code/__HELPERS/_lists.dm

```diff
@@ -22,7 +22,7 @@
 #define LAZYLEN(L) length(L)
 ///Sets a list to null
 #define LAZYNULL(L) L = null
-#define LAZYADDASSOC(L, K, V) if(!L) { L = list(); } L[K] += V;
+#define LAZYADDASSOC(L, K, V) if(!L) { L = list(); } if(!L[K]) {L[K] = list()} L[K] += V;
```

Comment:
Isn't this the same as `LAZYADDASSOCLIST`?

Reply:
originally done because i misunderstood what dm did with adding lists to nulls, now changed back to the current macro